### PR TITLE
allow to control active step when activeStepIndex change

### PR DESCRIPTION
**Pull request by groomain**
This PR allow you to control the active step outside the wizard.

### CODE REVIEW

**Comment by groomain** [comment]
Hello @mjangir any plan to merge these pull requests ?

**Comment by mjangir** [comment]
> Hello @mjangir any plan to merge these pull requests ?

Pipeline failed.

**Comment by groomain** [comment]
Sorry, I forgot to commit the import. Normally it should not failed now.

**Comment by mjangir** [comment]
@groomain Check line 18 if it is correct!

File: src/useWizard.ts [hunk 1]
```diff
@@ -12,6 +12,12 @@ const useWizard = (
   const isPrevDisabled: boolean = currentStep === 0;
   const isFirstStep: boolean = currentStep === 0;
   const isLastStep: boolean = currentStep >= total - 1;
+  useEffect(
+    () => {
+      setCurrentStep(activeStepIndex);
+    }
```
Comment (mjangir) [inline src/useWizard.ts:18]: There should be a comma here. Did you find it working in your local machine? I'm sure it didn't.

**Comment by groomain** [comment]
is it good for you?

**Comment by groomain** [comment]
The docs failed to be deployed. Is it because of the PR or juste a problem on github ?

**Pull request closed without merging**